OPHD/Things/Structures/StorageTanks.h

```diff
@@ -26,10 +26,27 @@ class StorageTanks : public Structure
 
 	StringTable createInspectorViewTable() override
 	{
-		StringTable stringTable(2, 1);
-
-		stringTable[{0, 0}].text = "Storage Capacity:";
-		stringTable[{1, 0}].text = std::to_string(storageCapacity());
+		StringTable stringTable(2, 5);
+
+		stringTable.setColumnText(
+			0,
+			{ 
+				"Storage Capacity",
+				ResourceNamesRefined[0],
+				ResourceNamesRefined[2],
+				ResourceNamesRefined[1],
+				ResourceNamesRefined[3],
```

Comment:
This use of out or order indexes concerns me. Is there a way we can fix this? I think I've seen this elsewhere in existing code too. Maybe we should re-order what the values mean if we typically use the values in a different order.